objection/utils/patchers/android.py

```diff
@@ -868,7 +869,7 @@ def zipalign_apk(self):
             self.required_commands['zipalign']['location'],
             '-p',
             '4',
-            self.apk_temp_frida_patched,
+            self.apk_temp_frida_patched if os.path.exists(self.apk_temp_frida_patched) else self.apk_source,
```

Comment:
This will modify the original APK, yeah? I would like to completely avoid modifying the source application.